v2/Dockerfile

```
@@ -3,14 +3,14 @@
 # check=error=true
 #
 #################################################
-FROM ghcr.io/opensafely-core/base-action:24.04 AS base-r
+FROM ghcr.io/opensafely-core/base-action:22.04 AS base-r
```

Lets use the BASE variable?
```suggestion
ARG BASE=22.04
FROM ghcr.io/opensafely-core/base-action:$BASE AS base-r
```